Fixes #18560 - Remove searchbar from 403, change 403 style

shlomizadok opened the pull request.

shlomizadok committed:
Fixes #18560 - Remove searchbar from 403, change 403 style

ohadlevy commented on app/views/common/403.html.erb:
```diff
@@ -1,11 +1,15 @@
-    :actions => link_to(_('Back'), main_app.root_path, :class => 'btn btn-default'),
-    :class => 'alert-danger',
-    :close => false %>
+<% content_for(:title, _("Permission denied")) %>
```
you can use 
```erb
<% title(..) %>
```
instead which will also set the page title and header.

shlomizadok replied:
Cowardly copied from `welcome` pages... :wink: 
Will change

dLobatog replied:
I can change this on merge, it's fine

ohadlevy commented:
@shlomizadok can you provide a screenshot ? thanks!

shlomizadok commented:
@rohoover  - May I have your take on the suggested styling here (I couldn't find any ref on @patternfly, so I have created my own...)

Currently the 403 page looks like:

![image](https://cloud.githubusercontent.com/assets/433583/23103888/b69a5482-f6cb-11e6-81b4-d67696f9f6f6.png)

In this PR, I have removed the search bar and changed the appearance to:

![image](https://cloud.githubusercontent.com/assets/433583/23103898/d8e33324-f6cb-11e6-9437-83c378d5c4bf.png)

Would love to hear your comments :smile:

dLobatog commented:
Thanks @shlomizadok  - I'll test changing the `content_for` to `title` on merge and will merge it if it works fine. I am not 100% convinced the empty state pattern should be used here, but after looking in various places (ManageIQ, patternfly-react-starter), I didn't see anyone using anything in particular for pages that deny actions due to permissions)

dLobatog commented:
`content_for` is fine, title adds also the title at the beginning which we don't want:

![screenshot from 2017-02-20 16-56-25](https://cloud.githubusercontent.com/assets/598891/23132361/a1a794a0-f78d-11e6-8d3a-790c6d87375b.png)

I'll merge for the moment as we will not get feedback from @rohoover until next week, this is an improvement already IMO, if more things have to be changed, that can be done later. 

@rohoover the merged version does not differ from the screenshot @shlomizadok posted

Rohoover commented:
@shlomizadok Nice!

Definitely no title @dLobatog 

![bitmoji](https://render.bitstrips.com/v2/cpanel/10215973-131321122_2-s4-v1.png?transparent=1&palette=1&width=246)
